python/ray/rllib/agents/sac/sac_policy_graph.py

```diff
@@ -0,0 +1,233 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from numbers import Number
+
+import numpy as np
+import tensorflow as tf
+
+from gym.spaces import Box
+
+import ray
+from ray.rllib.utils.error import UnsupportedSpaceException
+from ray.rllib.evaluation.tf_policy_graph import TFPolicyGraph
+from ray.rllib.utils.annotations import override
+
+from .models import GaussianLatentSpacePolicy, feedforward_model
+
+
+class SACPolicyGraph(TFPolicyGraph):
+    def __init__(self, observation_space, action_space, config):
+        if not isinstance(action_space, Box):
+            # TODO(hartikainen): Should we support discrete action spaces?
+            # I've seen several people requesting support for it.
```

Comment:
Would this be a matter of sampling from a categorical dist and also the associated entropy calculation? It seems like if get_action_dist() was used that might be automatically handled.

Reply:
Yeah, shouldn't be more complicated than switching the policy to categorical distribution. Awesome that `get_action_dist()` handles that out of the box!